### DISCUSSION TITLE
How to train a detectron2 model without bounding box in the training set but have the segmentation labels

### DISCUSSION THREAD

**Discussion by amruta5694** [facebookresearch/detectron2#2842] [Q&A]
Hello,
I have a training dataset which has many images but do not have bounding box annotations.
I have annotations for the images in validation and test dataset.
Goal is to train a detectron2 model using validation (with bbox) and training dataset(without bbox) without affecting the accuracy of the model.
When I tried to train the model with validation dataset only the accuracy was 71% but when I tried to combine both train and validation dataset the accuracy reduced to 9%.
Can someone please help me on how to train the model without the bounding box ?
